main/exercise/question.class.php

```diff
@@ -923,7 +919,7 @@ public function exportPicture($questionId, $courseInfo)
         $table = Database::get_course_table(TABLE_QUIZ_QUESTION);
         $sql = "UPDATE $table SET
                 picture = '".Database::escape_string($picture)."'
-                WHERE c_id = $course_id AND id='".intval($questionId)."'";
+                WHERE iid='".intval($questionId)."'";
```

Comment:
In this case we shouldn't need the quotes around the `iid = 'value'` because it's an integer.